prime-router/docs/design/design/topic.md

```diff
@@ -1 +1,61 @@
-TODO: Document ReportStream's Topic concept
\ No newline at end of file
+# Topic
+
+A topic consists of a name and a boolean indicating whether it's associated with the universal pipeline.  The primary
+intention is to silo data between different use cases.
+
+## Uses
+
+The primary use right now is to separate group of senders that route data through the universal pipeline and senders
+that route data through the legacy pipeline.  See below for more details.
+
+### Sender/Receivers
+
+- A sender has a configured topic and reports sent through the pipeline will be tagged with that topic
+- A receiver has configured topic and will only be routed reports that are tagged with that topic
+
+### Routing
+
+This is primary usage for topic and accomplishes the siloing, the routing step will only consider receivers that match
+the topic for the incoming messages.
+
+Additionally, each topic is configured with a different set of default filters that are applied if the receiver has not
+specifically defined their own.
+
+**TODO: remove the default filters associated with topics.  They are currently mostly unused within the universal pipeline and
+being linked to topics creates confusion and makes them less scalable.**
+
+### Translating
+
+The list of receivers generated from analyzing the `Endpoint` FHIR resources are filtered down to verify that the
+receivers topic is associated with the universal pipeline.
+
+**TODO: this [check](https://github.com/CDCgov/prime-reportstream/blob/ce91d6748aae94c5ab7c4cfc27da11c6d189521c/prime-router/src/main/kotlin/fhirengine/engine/FHIRTranslator.kt#L88) is likely unnecessary and can be removed**
```

Comment:
Agree that we can remove this. The check might shrink the window where if a topic of a receiver changes after routing we would use legacy batching for something that was routed for UP. But that's a very unlikely scenario anyway, and combined with the proposed change for batching, we would only consider topics during routing, which seems like the right move.